src/qml/pages/settings/SettingsStorage.qml

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../../controls"
+import "../../components"
+
+InformationPage {
+    Layout.fillWidth: true
+    bannerActive: false
+    bold: true
+    headerText: qsTr("Storage settings")
+    headerMargin: 0
+    detailActive: true
+    detailItem: ColumnLayout {
+        spacing: 0
+        StorageSettings {
```

Comment:
The naming here ends up being a bit funny as its nearly identical to the parent component's name. Maybe append to the component name something that signifies that it is just the selection piece or use "import as" when bringing in the component.

Reply:
Just noticed the naming here are the same words in reversed order :D, can anyone suggest better naming here?

@GBKS: `SettingsStorage.qml` represents the actualy page view for the Settings that are related to Storage. And `StorageSettings` is a component we place inside of `SettingsStorage.qml` that represents the table of advanced settings which are related to Storage.

Reply:
since this component is already under a "settings" folder.  Maybe StorageSettingsPage is better name for the component and then something equivalent for the other components since it is a "Page" that holds the StorageSettings